### ISSUE TITLE
Option to disable features

### ISSUE THREAD

**Issue by originalfoo** [simon56modder/ProceduralObjects#3]
Hi,

Would it be possible to add a mod option to disable certain features of the mod?

Specifically I only want to use it for text-editing (eg. [custom signs like these](https://steamcommunity.com/sharedfiles/filedetails/?id=2118779372)) but not for altering anything else.

**Comment by ErSpieltHD** [comment]
I agree with you @aubergine10, but wouldn't it make more sense to have an extra mod instead of disabling the features so it saves (work-)space?

I would use Mark-a-Route but it's outdated, not working and got removed from the Workshop and I can't code C# to remake it.
